### ISSUE TITLE
Self-hosted instance, user registration disabled, users can still register

### ISSUE THREAD

**Issue by Gauss23** [netbirdio/netbird#2749]
**Describe the problem**

Hi, just installed the newest self-hosted version with Zitadel 2.54.10. Disabled the user registration in Zitadel admin but users are still able to register.

**To Reproduce**

1. Install with getting-started script
2. login to Zitadel
3. disable user registration
4. the button to register is gone from the login page, but go to https://INSTANCE-URL/ui/login/register
5. and register
6. in Zitadel you'll see the user.

**Expected behavior**

User registration should stop working

**Are you using NetBird Cloud?**

No

**NetBird version**

`0.30.2`

**Comment by Gauss23** [comment]
Just check at Zitadel and it seems that this is a known bug:
https://github.com/zitadel/zitadel/issues/8123

**Comment by hnkl** [comment]
> Fix is released with [Zitadel 2.62.7](https://github.com/zitadel/zitadel/releases/tag/v2.62.7)
> 
> Is it safe to upgrade Zitadel?

i tested it in a lab environment. i don't see any difference after the update except that the registration no longer works.

I was still able to log in to Netbird and co.

**Comment by v1rusnl** [comment]
> > Fix is released with [Zitadel 2.62.7](https://github.com/zitadel/zitadel/releases/tag/v2.62.7)
> > Is it safe to upgrade Zitadel?
> 
> i tested it in a lab environment. i don't see any difference after the update except that the registration no longer works.
> 
> I was still able to log in to Netbird and co.

Can confirm this now. Upgraded to 2.64.1 with no issues.

**Comment by v1rusnl** [comment]
Latest 0.30.3 release still refers to Zitadel 2.54.10 in docker-compose.yml. Any reasons for that? I think [https://github.com/zitadel/zitadel/pull/8782](url) is a pretty important bugfix.

**Comment by mlsmaycon** [comment]
The Pr seems to be in an RC version only. Have you got the chance to test it?

**Comment by v1rusnl** [comment]
> The Pr seems to be in an RC version only. Have you got the chance to test it?

Correct, this was an RC. I tested it with 2.64.1 before it was shipped with Netbird 0.31.0. I'm not the issue opener but since 0.31.0 comes with 2.64.1., I guess the issue can be blosed.

**Comment by nazarewk** [comment]
Hello @Gauss23,

We're currently reviewing our open issues and would like to verify if this problem still exists in the [latest NetBird version](https://github.com/netbirdio/netbird/releases).

Could you please confirm if the issue is still there?

We may close this issue temporarily if we don't hear back from you within **2 weeks**, but feel free to reopen it with updated information.

Thanks for your contribution to improving the project!

**Comment by mlsmaycon** [comment]
closing issue due to no recent feedback. Feel free to open a new one if the issue persist or reopen if this was a feature request.
